### PR TITLE
twl_strpos + Norme

### PR DESCRIPTION
**Pull request by mgiraud**
Added twl_strpos function which return the position of a character in a string (same as php func) or -1 if the char is absent.
Norme for new version of norminette

### CODE REVIEW

**Comment by yyang42** [comment]
Cool, great, the norminette is working again!!

image

It seems that your local git email is not the same as the one set on github.
So your account is not really "linked". Same remark for @atamano
Can you fix that? 
http://stackoverflow.com/questions/750172/change-the-author-of-a-commit-in-git

File: tests/moulitest_tests/tests/twl_xstring/twl_strpos.spec.c [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+	str = strdup("abcdefghi");
+	mt_assert(twl_strpos(str, 'b') == 2);
+	mt_assert(twl_strpos(str, 'i') == 9);
+	mt_assert(twl_strpos(str, 0) == -1);
+	mt_assert(twl_strpos(str, 'p') == -1);
+	free(str);
+}
+
+void	suite_twl_strsplit(t_suite *suite)
```
Comment (yyang42) [inline tests/moulitest_tests/tests/twl_xstring/twl_strpos.spec.c]: Should be `void suite_twl_strpos(t_suite *suite)`

Comment (yyang42) [inline tests/moulitest_tests/tests/twl_xstring/twl_strpos.spec.c]: `twl_strpos` does not passed the tests you wrote :(

```
make check
>>>> twl_strpos.spec.c ----------- [FAIL] F [FAIL] simple_strpos -> twl_strpos(str, 'b') == 2
```

**Comment by mgiraud** [comment]
C'est bon, j'ai fix. #jecodeaveclecul

**Comment by yyang42** [comment]
:+1:

**Pull request merged**